src/ResourceManager/AnalysisServices/Commands.AnalysisServices/Commands/GetAzureRmAnalysisServicesServerGatewayStatus.cs

```diff
@@ -0,0 +1,53 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System.Management.Automation;
+using Microsoft.Azure.Commands.AnalysisServices.Models;
+using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+using Microsoft.Azure.Management.Analysis.Models;
+
+namespace Microsoft.Azure.Commands.AnalysisServices
+{
+    [Cmdlet(VerbsCommon.Get, "AzureRmAnalysisServicesServerGatewayStatus"),
+        OutputType(typeof(string))]
+    [Alias("Get-AzureAsGateway")]
+    public class GetAzureAnalysisServicesServerGatewayStatus: AnalysisServicesCmdletBase
```

Comment:
See cmdlet review.  This cmdlet must implement the ResourceId parameter set.  Also, why is this just returnign ststus rather than details of the gateway including status?